AudioLink/Scripts/ThemeColorController.cs

```diff
@@ -0,0 +1,147 @@
+﻿using UnityEngine;
+#if VRC_SDK_VRCSDK2 || VRC_SDK_VRCSDK3
+using VRC.SDKBase;
+#endif
+using UnityEngine.UI;
+
+namespace VRCAudioLink
+{
+    #if UDON
+        using UdonSharp;
+        using VRC.Udon;
+
+        [UdonBehaviourSyncMode(BehaviourSyncMode.Manual)]
+        public class ThemeColorController : UdonSharpBehaviour
+        {
+            [UdonSynced] private int themeColorMode;
+
+            [UdonSynced] public Color[] customThemeColors = new Color[4]{
```

Comment:
Wait, you can sync this? I wasn't aware o.o

Reply:
:D
From [Udon docs](https://docs.vrchat.com/docs/udon-networking#requestserialization):

> You can sync variables which are arrays for the following types:
> bool, char, byte, sbyte, short, ushort, int, uint, long, ulong, float, double, Vector2, Vector3, Vector4, Quaternion, Color and Color32.